### PR TITLE
Anti roomba league

### LINKED ISSUE #6437: Roomba cleaning up medevac beacons
## Reproduction:

Roomba is cleaning up medevac beacons. I don't think this is intentional.

### PR DESCRIPTION
**Pull request by InterroLouis**
## About The Pull Request

Adds a new item flag which prevents an item from being vacuumed up by the roomba. Not on any items by default, but gets added to medevac beacons when they're planted and removed from them when they're picked up. 
Fixes #6437 

## Why It's Good For The Game

No more roomba'd medevac beacons. Unless you leave it on the floor unplanted or someone unplants it and then ignores it. But that's just user error. 

## Changelog
:cl:
fix: Planted medevac beacons are now vacuum-proof. 
/:cl:

### CODE REVIEW

**Comment by lbnesquik** [comment]
Very good.

**Review by TiviPlus — APPROVED** [review]

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 1]
```diff
@@ -626,6 +626,7 @@ GLOBAL_LIST_EMPTY(activated_medevac_stretchers)
 		to_chat(user, "<span class='warning'>[src]'s interface is locked! Only a Squad Leader, Corpsman, or Medical Officer can unlock it now.</span>")
 		return
 	user.drop_held_item()
+	flags_item += NO_VACUUM
 	anchored = TRUE
```
Comment (TiviPlus) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:630]: This isnt how flags work

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm (hunk 1 above)
Comment (InterroLouis) [reply code/game/objects/structures/stool_bed_chair_nest/bed.dm:630]: Of course it's how flags work, otherwise it wouldn't have worked.

Comment (TiviPlus) [reply code/game/objects/structures/stool_bed_chair_nest/bed.dm:630]: Dont lecture me if you dont fix it ill just close this

Comment (InterroLouis) [reply code/game/objects/structures/stool_bed_chair_nest/bed.dm:630]: Then explain it to me.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**